src/main/java/com/waverleysoftware/Main.java

```diff
@@ -0,0 +1,90 @@
+package com.waverleysoftware;
+
+import com.waverleysoftware.facade.ShelterFacade;
+import com.waverleysoftware.model.Animal;
+
+import java.util.Scanner;
+
+import static io.vavr.API.*;
+import static org.apache.commons.lang3.math.NumberUtils.toInt;
+
+
+class Main {
+
+    private static void dogPicture(){
+        System.out.println("\n" +
```

Comment:
What is it?

Reply:
@maksimgerasimenko 
dogPicture method will print in console logo of Shelter.